diff -urN vqfplugin-0.9.orig/src/vqf.cxx vqfplugin-0.9/src/vqf.cxx
--- vqfplugin-0.9.orig/src/vqf.cxx	2001-05-06 19:38:05.000000000 +0000
+++ vqfplugin-0.9/src/vqf.cxx	2004-05-19 10:32:39.923225664 +0000
@@ -127,7 +127,7 @@
 /* Check if this file belongs to us
  * Easist is to check extension. 
  * Alternatively, check if the first few bytes of a file is TWINVQ */
-static int  vqfFile(char *filename)
+int  vqfFile(char *filename)
 {
   char *ext = strrchr(filename, '.');
   if (ext && !strcasecmp(ext, ".vqf")) return TRUE;
